app/src/lib/release-notes.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  ReleaseMetadata,
+  ReleaseNote,
+  ReleaseSummary,
+} from '../models/release-notes'
+
+const itemEntryRe = /^\[(new|fixed|improved|removed|added|pretext)\]\s(.*)/i
+
+const monthNames = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+]
+
+function getSuffix(day: number) {
+  const remainder = day % 10
+  if (remainder === 1) {
+    return 'st'
```

Comment:
What about day 11? That's going to be suffixed '11st'

Reply:
How about using a package for this, such as [`ordinal`](https://github.com/dcousens/ordinal)?

Reply:
Nice spot @saul, and good idea @j-f1!